### ISSUE TITLE
Ignoring the loss in some objects rotation

### ISSUE THREAD

**Issue by 123alaa** [xingyizhou/CenterNet#183]
Hi @xingyizhou ,
I am training 3d model, and for small object, I give them alpha = 0, it works fine. To make it better, I tried to change the BinRotLoss() in losses.py, to make zero in all the indexes of these objects. (i used torch.where by class_id, and make the output and the gt of these objects as zero, so their loss will be zero, i.e will not have weight in the loss). But It was not working. Is this should work? because you used cross entropy, should i gave them different values?

Attached the change in  BinRotLoss().

[losses.docx](https://github.com/xingyizhou/CenterNet/files/3355163/losses.docx)

**Comment by xingyizhou** [comment]
The code on RotLoss is copied from [another project](https://github.com/ucbdrive/3d-vehicle-tracking). I haven't investigated into that and not able to improve this.
